CHANGELOG.md

```diff
@@ -5,29 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [2.26.0] - 2022-01-14
     
 ### Added
     
 ### Removed
 
 ### Fixed
+- Fixed delays in advertising simulcast stream switches due to asynchronous and out of order checks
+- Fixed Firefox video tiles containing stale frames from previous transceivers by not attempting to reuse inactive transceivers. This switches to using `RTCRtpTransceiver.stop` when possible and may fix other incorrect tile bugs.
 - Fix the bug that the max bandwidth set by the chooseVideoInputQuality API is ignored when the Chime SDK retries the connection.
 - Added additional pausing of `MonitorTask` and `ReceiveVideoStreamIndexTask` to avoid modifying mutable state mid-subscribe
 
-### Changed
-
-## [2.26.0] - 2022-01-14
-    
-### Added
-    
-### Removed
-    
 ### Changed
 - Made `SimulcastUplinkObserver.encodingSimulcastLayersDidChange` (*not* `AudioVideoObserver.encodingSimulcastLayersDidChange`) synchronous.
-    
-### Fixed
```

Comment:
So the order right now (as automated added by script) is Added, Removed, Changed, Fixed so can we swap the Changed and Fixed part to be consistence :D